src/Tests/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/RazorSourceGeneratorTests.cs

```diff
@@ -37,12 +39,51 @@ public async Task SourceGenerator_RazorFiles_Works()
             var compilation = await project.GetCompilationAsync();
             var driver = await GetDriverAsync(project);
 
-            var result = RunGenerator(compilation!, ref driver);
+            var result = RunGenerator(compilation!, ref driver)
+                            .VerifyPageOutput(
+@"#pragma checksum ""Pages/Index.razor"" ""{ff1816ec-aa5e-4d10-87f7-6f4963833460}"" ""6b5db227a6aa2228c777b0771108b184b1fc5df3""
```

Comment:
FYI - Historically this was bothersome because the codegen would change for existing code and it would block insertion PRs and it was non trivial to get this string. The razor-compiler had an option to regen it but it’s a bit of a pain to use and makes diffs harder to work with.